package.json

```diff
@@ -38,80 +38,75 @@
     "prepare": "husky install"
   },
   "dependencies": {
-    "@nodevu/core": "^0.1.0",
-    "@types/node": "^18.16.19",
-    "@vcarl/remark-headings": "^0.1.0",
+    "@nodevu/core": "0.1.0",
+    "@types/node": "18.16.19",
```

Comment:
this is a devDependency right?

Reply:
Nope

Reply:
Technically, yes, but required by next.js to build.